<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>451</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>421</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QPushButton" name="Operador_Clear">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>130</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Solve">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Solve</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Operador_Delete">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;==</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number9">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineaDeComandos">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
background-image: url(:/imagenes/gambito1.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Number0">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>210</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Operador_Suma">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Operador_Multiplicacion">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Operador_Resta">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QPushButton" name="operador_Division">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number8">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="postfijo_LE">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>270</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>250</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Postfijo...... sin y cos se evaluan antes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Operador_elevar">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>250</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Operador_coseno">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>cos()</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Operador_seno">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>210</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>sen()</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Operador_cerrar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Operador_abrir">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>120</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Valide creo q todos lo errores</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>140</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Creo que esta completamente </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>160</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Funcional</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="otro">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>sin(199)cos(32)^2((2+32)/(3*4))</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Puede meterle esta funcion la respuesta es: -1.738728  puede comprobarlo en la sig pag: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="otra2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>330</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>http://www.wolframalpha.com/</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="grid"/>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>294</x>
        <y>110</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>y=</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>334</x>
        <y>12</y>
        <width>121</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Intervalos</string>
      </property>
      <widget class="QLineEdit" name="intervale1">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="intervalo2">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>46</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>de</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>51</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="graficar">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Graficar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="funcion_graf">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_rodas">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>31</width>
        <height>171</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>15</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>15</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>15</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>15</number>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>y</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
